linux_os/guide/services/ntp/service_timesyncd_configured/bash/shared.sh

```diff
@@ -20,6 +20,11 @@ do
     sed -i 's/^NTP/#&/g' "$current_cfg"
     sed -i 's/^FallbackNTP/#&/g' "$current_cfg"
 done
+# Create /etc/systemd/timesyncd.d if it doesn't exist
+if [ ! -d "/etc/systemd/timesyncd.d" ]
+then 
+    mkdir etc/systemd/timesyncd.d
```

Comment:
```suggestion
    mkdir /etc/systemd/timesyncd.d
```

Reply:
Please add the full path to the `mkdir` call.

Reply:
Fixed, my bad